Gems/LmbrCentral/Code/Source/Ai/NavigationComponent.cpp

```diff
@@ -9,7 +9,6 @@
 #include "EditorNavigationUtil.h"
```

Comment:
Probably everything in LmbrCentral related to AI/Navigation can be removed at this point, but maybe better to tackle that as a separate/self-contained effort.